astropy/io/votable/tests/table_test.py

```diff
@@ -67,7 +67,9 @@ def test_table(tmpdir):
         if 'arraysize' in d:
             assert field.arraysize == d['arraysize']
 
-    writeto(votable2, os.path.join(str(tmpdir), "through_table.xml"))
+    # W39: Bit values can not be masked
+    with pytest.warns(W39):
```

Comment:
Nothing much I can do here without scope creep. I think it is correct behavior for writer to be more verbose than reader. But I have no desire to fix the data file so that it stops complaining about bit values and masking.